src/Components/test/testassets/ComponentsApp.Server/wwwroot/index.html

```diff
@@ -9,8 +9,46 @@
     <link href="css/site.css" rel="stylesheet" />
 </head>
```

Comment:
At some point I expect we'll want to put a default built-in reconnection UI into `components.server.js`, so it does something useful even without developers knowing about circuit handlers. Perhaps we could even look at doing that immediately after this PR goes in, because otherwise the benefits of reconnection are lost for all but the most advanced of developers.

Reply:
Do you have any thoughts on what this would be like? I was hoping we would do the UI as part of the templates. This gives us the flexibility of doing something nice (rather than functional) while allowing users to customize in a fairly straightforward way.